### PR TITLE
allow for shimming with minimal changes

### PR DESCRIPTION
**Pull request by swanderz**

### CODE REVIEW

File: macros/schema_tests/_generalized/expression_is_true.sql [hunk 1]
```diff
@@ -29,7 +37,7 @@ with grouped_expression as (
         {{ group_by_column }} as col_{{ loop.index }},
         {% endfor -%}
         {% endif %}
-        {{ expression }} as expression
+        {{ dbt_expectations.truth_expression(expression) }}
```
Comment (clausherther) [inline macros/schema_tests/_generalized/expression_is_true.sql:40]: Good idea! Might be to add a comment to `truth_expression` why we need that. Without knowing that you're shimming this for another package, readers (i.e. my future self) might be confused.

File: macros/schema_tests/aggregate_functions/expect_column_most_common_value_to_be_in_set.sql [hunk 2]
```diff
@@ -30,7 +30,12 @@ with value_counts as (
     {% if row_condition %}
     where {{ row_condition }}
     {% endif %}
-    group by 1
+
+    group by {% if quote_values -%}
+                {{ column_name }}
+            {%- else -%}
+                cast({{ column_name }} as {{ data_type }})
```
[the remark this answers is not in the document]
Comment (clausherther) [reply macros/schema_tests/aggregate_functions/expect_column_most_common_value_to_be_in_set.sql:37]: Nm, just looked at the rest of my code and you just copied that. I never like repeating something like that. I wonder if we should cast or not cast in a separate earlier CTE and then simply group on the result in a second one. What do you think?
Maybe something like
```sql
with value_fields as (

    select
        {% if quote_values -%}
        {{ column_name }}
        {%- else -%}
        cast({{ column_name }} as {{ data_type }})
        {%- endif %} as value_field,
        1 as value_count

    from {{ model }}
    {% if row_condition %}
    where {{ row_condition }}
    {% endif %}

),
value_counts as (

    select
        value_field,
        sum(value_count) as value_count

    from value_fields
    {% if row_condition %}
    where {{ row_condition }}
    {% endif %}
    group by 
        value_field

),
...
```

[the remark this answers is not in the document]
Comment (swanderz) [reply macros/schema_tests/aggregate_functions/expect_column_most_common_value_to_be_in_set.sql:37]: great point. @alieu and I also toyed with this idea and are happy to take this approach.

**Comment by clausherther** [comment]
Hi @swanderz just wanted to revive this one - anything you need from me to move forward here? 
I'd like to close all the outstanding PRs before dbt 0.20 comes out, since that will require a refactor of the test code.

**Comment by swanderz** [comment]
closing in favor of #64

**Pull request closed without merging**